### PR TITLE
Create wrapper around AudioPlayer for Google Play

### PR DESCRIPTION
**Pull request by L1fescape**
Also Move the sign in to google form to library tab and load your google play favorites into the song list below the sign in form.

### CODE REVIEW

File: src/actions/google.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    });
+
+    fetch(endpoint)
+      .then(res => res.json())
+      .then((response) => {
+        dispatch({
+          type: actionTypes.GPLAY_SIGNIN_SUCCESS
+        });
+
+        let songs = formatSongResponse(response);
```
Comment (likethemammal) [inline src/actions/google.js]: make songs a const

File: src/components/player/index.js [hunk 2]
```diff
@@ -37,6 +38,19 @@ class Player extends Component {
           dispatchPlaySong={dispatchPlaySong}
           dispatchPauseSong={dispatchPauseSong}
         /> }
+      { song.google &&
+        <GooglePlayer
+          song={song}
+          isPlaying={isPlaying}
+          volume={volume}
+          seekPercent={seekPercent}
```
Comment (likethemammal) [inline src/components/player/index.js]: I know you're copy pasting, but this is cleaner as 

```
<GooglePlayer
    {...{
        song,
        isPlaying,
        volume,
        etc,
    }}
/>
```

File: src/components/song-list/type-cell.js [hunk 3]
```diff
@@ -12,8 +12,8 @@ const TypeCell = ({rowIndex, data, columnKey, ...props}) => {
     songType = "youtube"
   } else if (song.soundcloud) {
     songType = "soundcloud"
-  } else if (song.url) {
-    songType = "music"
+  } else if (song.google) {
```
Comment (likethemammal) [inline src/components/song-list/type-cell.js]: this block should get refactored at some point

Comment (L1fescape) [inline src/components/song-list/type-cell.js]: Yeah I agree

File: src/components/player/index.js (hunk 2 above)
Comment (L1fescape) [inline src/components/player/index.js]: oooh nice. I haven't seen that style before.

Comment (likethemammal) [inline src/components/player/index.js]: yea, removes the need to copy paste prop names. makes it easier to send in objects too

Comment (likethemammal) [inline src/components/player/index.js]: actually all 3 of the player components could receive the same object and just have song changed, would get rid of a bunch of extra

**Pull request merged**